Docs: Clarify running `make install` when changing dependencies (#7944)
plone/volto: docs/source/contributing/acceptance-tests.md

It's recommended to start three individual terminal sessions, one each for running the Plone backend, the Volto frontend, and the acceptance tests.
All sessions should start from the `packages/volto` directory.

> If you have changed any dependency, run `make install` from the repository root and commit the updated {file}`pnpm-lock.yaml` first; otherwise acceptance tests may fail due to an out-of-date lockfile.

Reviewer: ```suggestion
If you've changed any dependency, run `make install` from the repository root and commit the updated {file}`pnpm-lock.yaml` lockfile.
Otherwise acceptance tests may fail due to an out-of-sync lockfile.
```

> See {ref}`developing-core-when-changing-dependencies-label`.

Reviewer: ```suggestion
See {ref}`developing-core-dependency-updates-label`.
```
